src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Current.net45.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Diagnostics
 {
+    [Serializable]
```

Comment:
This is for 4.5 only as declared in  Activity.Current.net45.cs (that is picked up for net45 target only).